bluesky/run_engine.py

```diff
@@ -876,6 +891,13 @@ def _close_run(self, msg):
 
     @asyncio.coroutine
     def _create(self, msg):
+        """Trigger the run engine to start collecting data for an event
```

Comment:
Pardon a suggestion about the wording here. This is fine, but since elsewhere in the codebase we use the word "bundling" it might help to include that here -- "...to bundle future readings into an Event document" for example.